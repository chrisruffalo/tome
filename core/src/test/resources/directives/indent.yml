%{ include 'fragments/insert.yml' indent = 0}%

child:
  %{ include 'fragments/insert.yml' indent = 2}%

grandchild:
  great:
    %{ include 'fragments/insert.yml' indent = 4}%

bad:
  %{ include 'fragments/insert.yml' indent = 2 indent_first}%