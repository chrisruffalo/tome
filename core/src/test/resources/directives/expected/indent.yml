key: value
map:
  key0: value
  key1: value
list:
  - item1
  - item2

child:
  key: value
  map:
    key0: value
    key1: value
  list:
    - item1
    - item2

grandchild:
  great:
    key: value
    map:
      key0: value
      key1: value
    list:
      - item1
      - item2

bad:
    key: value
  map:
    key0: value
    key1: value
  list:
    - item1
    - item2
